core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceImpl.java

```diff
@@ -243,27 +259,48 @@ public TranslationScope getFinalScope(String strTranslationJobID) throws Transla
 
     @Override
     public TranslationStatus updateTranslationJobState(String strTranslationJobID, TranslationState state)
-        throws TranslationException {
-    	if(strTranslationJobID=="dummy"){
-    		log.debug("Dummy Translation job detected");
+      throws TranslationException {
+    	if(strTranslationJobID=="lilt"){
+    		log.debug("Lilt Translation job detected");
     	} else if(strTranslationJobID == null) {
     		log.debug("Job was never sent to TMS. Updated using Export/Import");
     	} else {
-    		bootstrapTmsService.setTmsProperty(strTranslationJobID, BootstrapTmsConstants.BOOTSTRAP_TMS_STATUS, state.getStatus().toString());
-    		log.warn("JOB ID is {}",strTranslationJobID);
+    		// bootstrapTmsService.setTmsProperty(strTranslationJobID, BootstrapTmsConstants.BOOTSTRAP_TMS_STATUS, state.getStatus().toString());
     	}
     	if(state.getStatus() == TranslationStatus.COMMITTED_FOR_TRANSLATION) {
     		log.trace("Uploaded all Translation Objects in job {}",strTranslationJobID);
     	}
-        return state.getStatus();
+      return state.getStatus();
     }
 
     @Override
     public TranslationStatus getTranslationJobStatus(String strTranslationJobID) throws TranslationException {
         log.trace("BootstrapTranslationServiceImpl.getTranslationJobStatus");
-        String status = bootstrapTmsService.getTmsJobStatus(strTranslationJobID);
-        log.debug("Status for Job {} is {}", strTranslationJobID, status);
-        return TranslationStatus.fromString(status);
+        try {
+          int imported = 0;
+          int translated = 0;
+          SourceFile[] files = liltApiClient.getFiles(strTranslationJobID);
+          // loop backwards through the list since the most recent files will be at the end.
+          for (SourceFile file : files) {
+            if (file.labels.contains("status=IMPORTED")) {
+              imported++;
+            }
+            if (file.labels.contains("status=TRANSLATED")) {
+              translated++;
+            }
+          }
```

Comment:
There's a stream idiom you could use to write this more concisely:
```
int imported = files.stream().filter(f -> f.labels.contains("status=IMPORTED")).count();
```

But since you're computing two values, I don't think there's a good way to use that without iterating over the loop twice.  So what you have is fine.